src/js/swarmAnimation.js

```diff
@@ -7,18 +7,28 @@
  * functions for applying css classes, doing transforms consistently
  */
 
-const swarmAnimation = {
-	hide(el) {
-		el.classList.add('anim-fade--out');
-	},
+const DURATION_TYPE = {
+	short: 'short',
+	medium: 'medium',
+	long: 'long'
+};
 
-	show(el) {
-		el.classList.remove('anim-fade--in');
+const swarmAnimation = {
```

Comment:
It's a little unclear if these `swarmAnimation` functions are intended for documentation only, or if consumer apps should use them directly to enable animations.